jetbrains-core/resources/telemetryOverride.json

```diff
@@ -33,6 +39,18 @@
                     "type": "experimentState"
                 }
             ]
+        },
+        {
+            "name": "aws_toolInstallation",
```

Comment:
future-proof by adding an automatic vs manual invocation?

Reply:
what does that mean?

Reply:
we don't expose it now, but i assumed eventually we'd want to allow users to manually trigger install/update. can definitely tackle the telemetry definition for that when it's actually implemented

Reply:
I guess add two metadata flags: `automatic` and `updated`.
Though `passive` covers the manual vs. automatic right?

Reply:
the primary question I'm trying to answer here - is how many automated installations fail - this is something we can (potentially fix)

Reply:
okay, we can keep as-is for now then